### DISCUSSION TITLE
Calculate nearest Point on LineString from given Point

### DISCUSSION THREAD

**Discussion by marcellangmaier** [NetTopologySuite/NetTopologySuite#487] [Q&A]
First of all, thank you for this great library.

How can I calculate the nearest Point (Q) on a Linestring from a given Point (P).
I am working with Latitudes and Longitudes.

![point_q](https://user-images.githubusercontent.com/26068556/90411633-5a2b3800-e0ac-11ea-9e00-bb6f59d8c38d.png)

**Comment by FObermaier** [comment]
Have a look at the [ClosestPoints](https://github.com/NetTopologySuite/NetTopologySuite/blob/develop/test/NetTopologySuite.Samples.Console/Operation/Distance/ClosestPointExample.cs) example.
